README.md

```diff
@@ -15,24 +14,19 @@ Easy [Prismic](https://prismic.io/) rendering in [Ember.js](https://emberjs.com)
 <Prismic::Dom @nodes={{@prismicData}} />
 ```
 
-Compatibility
-------------------------------------------------------------------------------
+## Compatibility
 
-* Ember.js v3.16 or above
-* Ember CLI v2.13 or above
-* Node.js v10 or above
+- Ember.js v4.4 or above
+- Ember CLI v4.4 or above
+- Node.js v14 or above
```

Comment:
Isn't supposed to be node 16? Since 14 is already end of life

Reply:
Will be fixed by: https://github.com/qonto/ember-prismic-dom/pull/17

Reply:
@dbendaou Ok to drop Node 14 in a separate PR?

Reply:
As agreed with Djamel, this can be resolved in favour of https://github.com/qonto/ember-prismic-dom/pull/17

Reply:
Awesome!